.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,139 @@
+---
+name: Bug Report
+description: Create a report to help us improve.
+labels: [bug]
+assignees: aio-libs/triagers
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Thanks for taking a minute to file a bug report!**
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+      _Please fill out the form below with as many precise
+      details as possible._
+
+      [issue search]: ../search?q=is%3Aissue&type=issues
+
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >-
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: >-
+      Describe the steps to reproduce this bug.
+    value: |
+      1. Have certain environment
+      2. Run given code snippet in a certain way
+      3. See some behavior described
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Expected behavior
+    description: >-
+      A clear and concise description of what you expected to happen.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Logs/tracebacks
+    description: >-
+      |
+      If applicable, add logs/tracebacks to help explain your problem.
+      Paste the output of the steps above, including the commands
+      themselves and their output/traceback etc.
+    render: Python traceback
```

This should be a supported code highlighting "language", not free-form text.
```suggestion
    render: python-traceback
```